### PR TITLE
Metrics part 2

### PR DESCRIPTION
**Pull request by paolocappelletti**
Updates to docker & build variables to optionally expose /metrics endpoint on a separate port and with separate authentication password than the one used in rest API

Jira Ticket
[SDK-1793](https://horizenlabs.atlassian.net/browse/SDK-1793)

### CODE REVIEW

File: dockerfiles/evmapp/entrypoint.sh [hunk 1]
```diff
@@ -264,6 +270,14 @@ if [ -n "${SCNODE_REST_PASSWORD:-}" ]; then
 fi
 export SCNODE_REST_APIKEYHASH
 
+# set Metrics API password hash
+if [ "${SCNODE_METRICS_ENABLED:-}" = "true" ]; then
```
Comment (otoumas) [inline dockerfiles/evmapp/entrypoint.sh]: As the default is set above `:-false` there is no need to use `:-` here as there will always be a value in he env var

**Review by otoumas — APPROVED** [review]
LGTM Just a non blocking comment

File: dockerfiles/evmapp/entrypoint.sh [hunk 2]
```diff
@@ -27,6 +27,12 @@ LOG4J_CUSTOM_CONFIG=""
 SCNODE_REMOTE_KEY_MANAGER_ENABLED="${SCNODE_REMOTE_KEY_MANAGER_ENABLED:-false}"
 export SCNODE_REMOTE_KEY_MANAGER_ENABLED
 
+SCNODE_METRICS_ENABLED="${SCNODE_METRICS_ENABLED:-false}"
+export SCNODE_METRICS_ENABLED
```
Comment (ysibirski) [inline dockerfiles/evmapp/entrypoint.sh]: I would combine line 31 and 33 in one line like first define 2 vars and then
```
export SCNODE_METRICS_ENABLED SCNODE_METRICS_PORT
```

**Review by ysibirski — APPROVED** [review]
LGTM, one non blocking comment as well

**Review by otoumas — APPROVED** [review]
LGTM

**Review by saratnt — APPROVED** [review]
LGTM

**Pull request merged**